make-release.sh

```diff
@@ -29,6 +29,7 @@ init() {
   BUILDX_PLATFORMS="linux/amd64,linux/ppc64le"
   DEV_WORKSPACE_CONTROLLER_VERSION="main"
   DEV_WORKSPACE_CHE_OPERATOR_VERSION="main"
+  STABLE_CHANNELS="stable-all-namespaces,stable"
```

Comment:
dumb question... if you're only using this variable as a bash for-loop, why use a comma and then have to translate the comma to a space?

why not just do...

```
STABLE_CHANNELS="stable-all-namespaces stable"
```

Reply:
That is a good point :D I will change it